qemu/tests/cfg/qemu_guest_agent.cfg

```diff
@@ -126,7 +126,20 @@
             Windows:
                 # Windows guest only works with qxl driver.
                 vga = "qxl"
+        - check_hotplug_virtio_console:
+            only default
+            gagent_check_type = hotplug_virtio_console
+            extra_params += "-device virtio-serial-pci,id=virtio-serial0,bus=pci.0,addr=0x7"
+            backend_char_plug = "socket"
+            id_char_plug = "serial0"
+            path_char_plug = "/tmp/org.qemu.guest_agent.0"
+            hotplug_gagent_serial_type = virtio
+            dev_driver = "virtserialport"
+            dev_id = "serial0"
+            dev_name = "org.qemu.guest_agent.0"
     variants:
+        - @default:
+            only check_hotplug_virtio_console
```

Comment:
why not delete this variant, and add "only virtio_serial"  in  check_hotplug_virtio_console ?

Reply:
The hotuplug case has not relationship with virtio_serial and isa_serial. And hotplug/unplug guest agent device without any additional serial in the case.